framework_tvb/tvb/interfaces/web/controllers/simulator/simulator_controller.py

```diff
@@ -259,7 +272,11 @@ def set_model(self, **data):
         session_stored_simulator, is_simulation_copy, is_simulation_load, _ = self.context.get_common_params()
 
         if cherrypy.request.method == POST_REQUEST:
-            self.context.add_last_loaded_form_url_to_session(SimulatorWizzardURLs.SET_MODEL_PARAMS_URL)
+            set_next_wizard = True
+            if SimulatorController.KEY_KEEP_SAME_SIM_WIZARD in data:
+                set_next_wizard = False
+            if set_next_wizard:
```

Comment:
Condition could be based on:
keep_same_wizard = `SimulatorController.KEY_KEEP_SAME_SIM_WIZARD in data`
if keep_same_wizard is False: ...